packages/store/src/StoreCore.sol

```diff
@@ -14,9 +14,9 @@ import { IStoreHook } from "./IStore.sol";
 
 library StoreCore {
   // note: the preimage of the tuple of keys used to index is part of the event, so it can be used by indexers
-  event MudStoreSetRecord(bytes32 table, bytes32[] key, bytes data);
-  event MudStoreSetField(bytes32 table, bytes32[] key, uint8 schemaIndex, bytes data);
-  event MudStoreDeleteRecord(bytes32 table, bytes32[] key);
+  event StoreSetRecord(bytes32 table, bytes32[] key, bytes data);
+  event StoreSetField(bytes32 table, bytes32[] key, uint8 schemaIndex, bytes data);
+  event StoreDeleteRecord(bytes32 table, bytes32[] key);
```

Comment:
It's a bummer we can't import these from `IStore` or the reverse so they can be defined in one place.